README.rst

```diff
@@ -45,11 +55,39 @@ upon by some of the world's leading scientists and engineers. If you need to
 manipulate numbers on a computer and display or publish the results, give
 SciPy a try!
 
-For the installation instructions, see INSTALL.rst.txt_.
+For the installation instructions, see `our install
+guide <https://scipy.github.io/devdocs/getting_started.html#installation>`__.
+
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. Have a look at `our contributing
+guide <http://scipy.github.io/devdocs/dev/hacking.html>`__.
+
+Writing code isn’t the only way to contribute to SciPy. You can also:
+
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve `our website <https://github.com/scipy/scipy.org>`__
+- develop graphic design for our brand assets and promotional materials
+- translate website content
```

Comment:
Not trying to discourage it, but NumPy seems to have a bit of discussion about the amount of work it is to maintain translations: https://numpy.org/neps/nep-0028-website-redesign.html#translation-multilingual-i18n

I wonder if they ended up sticking with one of the rules listed there: 

> The language must have a dedicated maintainer

Ralf would probably know--I've seen them on a bunch of the language PRs.

Reply:
Oh yes, this should be removed here. We will translate the NumPy website, but not the docs. And for SciPy we first need a new website ....

Reply:
Ok sure I will remove it, thanks